src/pages/ServiceList/__tests__/ServiceListComponent.test.ts

```diff
@@ -1,15 +1,15 @@
+import { ServiceHealth, RequestHealth, WithHealth } from '../../../types/Health';
 import { ServiceListItem } from '../../../types/ServiceList';
-import { ServiceHealth, RequestHealth } from '../../../types/Health';
 import { ServiceListFilters } from '../FiltersAndSorts';
 
-const makeService = (name: string, errRatio: number): ServiceListItem & { health: ServiceHealth } => {
+const makeService = (name: string, errRatio: number): WithHealth<ServiceListItem> => {
   const reqErrs: RequestHealth = { errorRatio: errRatio, inboundErrorRatio: errRatio, outboundErrorRatio: -1 };
+  const health = new ServiceHealth(reqErrs, { rateInterval: 60, hasSidecar: true });
+
   return {
     name: name,
-    health: new ServiceHealth(reqErrs, { rateInterval: 60, hasSidecar: true })
-  } as ServiceListItem & {
-    health: ServiceHealth;
-  };
+    health: health
+  } as any;
```

Comment:
I think I would prefer to set empty values for the other properties not being used to avoid casting as any.

Reply:
Yeah, that's probably what I'm going to do, but I don't like it very much, adds information to the test that is not under subject of testing and make things messy in general, but I think that's the best option.

Reply:
In this situation I prefer simpler / more readable test than type accuracy... so as Caina did. It's subject to debate...

Reply:
I think we miss the classes from other languages which set the defaults in constructors and then you only set the data you need ;)